src/CommunityToolkit/Xamarin.CommunityToolkit/Behaviors/Animations/AnimationTypes/RotateAnimation.shared.cs

```diff
@@ -26,5 +26,59 @@ public override async Task Animate(View view)
 			await view.RotateTo(Rotation, Duration, Easing);
 			view.Rotation = 0;
 		}
+    }
+
+	public class RelRotateAnimation : AnimationBase
+    {
+        public static readonly BindableProperty RotationProperty =
+            BindableProperty.Create(nameof(Rotation), typeof(double), typeof(RelRotateAnimation), default(double),
+                BindingMode.TwoWay, null);
+
+        public double Rotation
+        {
+            get => (double)GetValue(RotationProperty);
+            set => SetValue(RotationProperty, value);
+        }
+
+        protected override uint DefaultDuration { get; set; } = 200;
+
+        public override async Task Animate(View view) =>
+            await view.RelRotateTo(Rotation, Duration, Easing);
+	}
+
+	public class RotateXAnimation : AnimationBase
+    {
+        public static readonly BindableProperty RotationProperty =
+            BindableProperty.Create(nameof(Rotation), typeof(double), typeof(RotateXAnimation), default(double),
+                BindingMode.TwoWay, null);
+
+        public double Rotation
+        {
+            get => (double)GetValue(RotationProperty);
+            set => SetValue(RotationProperty, value);
+        }
+
+        protected override uint DefaultDuration { get; set; } = 200;
+
+        public override async Task Animate(View view) =>
+            await view.RotateXTo(Rotation, Duration, Easing);
```

Comment:
All of the other `override Animate` methods push to the main thread. Should this push to the main thread too?

```csharp
public override Task Animate(View view) => Device.InvokeOnMainThreadAsync(() => view.RotateXTo(Rotation, Duration, Easing));
```